src/components/playbackManager.ts

```diff
@@ -167,19 +176,29 @@ export class playbackManager {
         );
     }
 
-    playMediaSource(playSessionId, item, mediaSource, options) {
+    // TODO eradicate any
+    playMediaSource(
+        playSessionId: string,
+        item: BaseItemDto,
+        mediaSource: any,
```

Comment:
I think this one can be set to the MediaSourceInfo type if I remember correctly

Reply:
@hawken93 Cool! Remember to change this before merging please.